### ISSUE TITLE
Suggestion: My _first_ heading is the toc, the _second_ is the filename

### ISSUE THREAD

**Issue by merlinuwe** [dvcrn/obsidian-filename-heading-sync#69]
Hi would you like to see an option to set the syc to the second # in a note.

**Comment by altroche** [comment]
Yes, I would need this so much! I normally use the second heading as filename, something that is currently not supported by Filename Heading Sync.

**Comment by dvcrn** [comment]
Should be fairly trivial to implement, adjust https://github.com/dvcrn/obsidian-filename-heading-sync/blob/master/main.ts#L278 to take some parameter which heading to find

Sadly I'm currently loaded with other projects and don't know yet when I can get around to add this
